packages/storybook/stories/controls.ts

```diff
@@ -0,0 +1,34 @@
+enum CONTROL_CATEGORY {
+  accessibility = 'Accessibility',
+  design = 'Design',
+  general = 'General',
+  slot = 'Slot',
+}
+
+const orderedControlCategories = [
+  CONTROL_CATEGORY.design,
+  CONTROL_CATEGORY.slot,
+  CONTROL_CATEGORY.general,
+  CONTROL_CATEGORY.accessibility,
+];
+
+function orderControls(control: Record<string, any>) {
+  try {
+    return Object.entries(control)
+      .sort((a, b) => {
+        return orderedControlCategories.indexOf(a[ 1 ].category) - orderedControlCategories.indexOf(b[ 1 ].category);
+      })
+      .reduce((res, entry) => {
```

Comment:
```suggestion
      .reduce((res, { key, value }) => {
```

Reply:
entry is an array containing [<prop>, <value>] not an object